app/pubspec.yaml

```diff
@@ -5,6 +5,8 @@ description: The pub.dartlang.org website.
 environment:
   sdk: ">=1.25.0-dev.9.0 <2.0.0"
 dependencies:
+  _popularity:
+    path: ../popularity
```

Comment:
Could we be consistent regarding:
  * package name and directory name, use `_popularity` everywhere
  * comments and location, use `pkg/_popularity` everywhere (and move it into subfolder, it's better if all side packages are in `pkg/`)

Reply:
Done